azure/types.go

```diff
@@ -372,6 +375,27 @@ type AADProfile struct {
 	AdminGroupObjectIDs []string
 }
 
+// LoadBalancerProfile - Profile of the cluster load balancer.
+type LoadBalancerProfile struct {
+	// ManagedOutboundIPs - Desired managed outbound IPs for the cluster load balancer.
+	ManagedOutboundIPs *int32
```

Comment:
nit: a slightly clearer description of why you'd use this vs outboundIPs might be nice. It's fairly clear from the API one is managed by AKS and one is BYO, but would be nice to be explicit.